### ISSUE TITLE
Error on execute scala-cli 

### ISSUE THREAD

**Issue by mobilemindtech** [VirtusLab/scala-cli#2331]
Hi,

I installed scala-li, and when running I got this error:

```shell
└─$ scala-cli compile .
[error] ./Main.scala:1:15: Expected primitive value: string, numeric or boolean but found identifier: com. Wrapping identifier in quotes usually solves the problem.
[error] //> using dep com.lihaoyi:upickle:3.1.2
[error]               ^
[error] ./Main.scala:1:15: Expected new line after the using directive, in the line; but found identifier: com
[error] //> using dep com.lihaoyi:upickle:3.1.2

```

Main.scala

```scala
//> using dep com.lihaoyi::upickle:3.1.2

import upickle.default.{ReadWriter}

case class Test(id: Int, date: Date) derives ReadWriter
```

env 

```shell
Scala CLI version: 0.1.18
Scala version (default): 3.2.1
                                                                                                                                                              
java 20.0.1 2023-04-18
Java(TM) SE Runtime Environment (build 20.0.1+9-29)
Java HotSpot(TM) 64-Bit Server VM (build 20.0.1+9-29, mixed mode, sharing)

```

**Comment by tgodzik** [comment]
Thanks for reporting! You scala-cli version seems to be older, which didn't support the quoteless syntax. Could you try updating?

How did you install Scala CLI? It might be possible that the package manager you used is not being updated actively (though I think all the ones listed on our website should be fine)

**Comment by mobilemindtech** [comment]
I installed the bit for the first time using the command suggested in the documentation: `curl -sSLf https://virtuslab.github.io/scala-cli-packages/scala-setup.sh | sh`

**Comment by tgodzik** [comment]
I see that the default version in the setup script is correct, did you install it recently or a while ago? You might want to try and reinstall if that was a while ago.

**Comment by lwronski** [comment]
@mobilemindtec Can you try running:
```
scala-cli update
```

This should update your `scala-cli` to the latest version.

**Comment by mobilemindtech** [comment]
@lwronski  when running the command `scala-cli update` i get the following message

```shell
Scala CLI was not installed by the installation script, please use your package manager to update scala-cli.
```

@tgodzik This error happened the day I installed it and on the same day I opened the issue. I tried to reinstall using the same command and got the same error:

```shell
curl -sSLf https://virtuslab.github.io/scala-cli-packages/scala-setup.sh | sh
```

**Comment by tgodzik** [comment]
Hmm.... I can't reproduce it, are you sure you don't have it installed previously and manually added to PATH? Could you check where it's installed and see if it's actually the one being installed with the script?

**Comment by mobilemindtech** [comment]
I found the problem. scala-cli comes installed with coursier. My Coursier was out of date and wouldn't run Coursier update, so I had to force install again, and then scala-cli was updated and it worked.

Weird that before updating coursier, I removed scala-cli using cs uninstall and tried to install using the previously mentioned script, but the command `scala-cli` was not being recognized after installation. This must have something to do with the coursier.

Thanks for the help and sorry for the inconvenience.
